src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -913,6 +913,16 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                 $output.ToString()
                             }
 
+                            # Add back original typename and remove PSExtendedError
+                            if ($_.PSObject.TypeNames.Contains('System.Management.Automation.ErrorRecord#PSExtendedError')) {
+                                $_.PSObject.TypeNames.Add('System.Management.Automation.ErrorRecord')
+                                $null = $_.PSObject.TypeNames.Remove('System.Management.Automation.ErrorRecord#PSExtendedError')
+                            }
+                            elseif ($_.PSObject.TypeNames.Contains('System.Exception#PSExtendedError')) {
```

Comment:
elseif? Is it possible for typenames to contain _both_ Exception#PSExtendedError _and_ ErrorRecord#PSExtendedError

Reply:
Based on current code, it is not possible as Get-Error only adds one or (else) the other.